src/hero.tsx

```diff
@@ -147,41 +149,41 @@ const Hero = React.forwardRef<HTMLDivElement, HeroProps>(
 
         if (isTabletOrSmaller) {
             return (
-                <div style={applyCssVars({[mediaStyles.vars.mediaBorderRadius]: '0px'})}>
```

Comment:
See this changes hiding whitespaces (https://github.com/Telefonica/mistica-web/pull/1106/files?diff=split&w=1).

Wrapping the content with this div was causing `height: "100%"` to not work. I've moved this css var a bit down to avoid the unnecessary extra div and now it works as expected